src/Stan/Report/Html.hs

```diff
@@ -38,12 +49,55 @@ stanMain an = main_
       # divIdClassH "Analysis Info" "" ()
       )
     # divIdClassH "Graphs" "" (p_ "Maybe later")
-    # divIdClassH "Observations" "" (pre_ an)
+    # divIdClassH "Observations" "" (stanObservations an)
     # divIdClassH "Configurations" "" (p_ "Later")
     # divIdClassH "Summary" "" (p_ "Later")
     # divIdClassH "Inspections" "" (p_ "Later")
     )
 
+stanObservations Analysis{..} =
+    map stanPerFile
+    $ filter (not . null . fileInfoObservations)
+    $ Map.elems analysisFileMap
+
+stanPerFile FileInfo{..} = divIdClass "" ""
+    ( h3_ (toText fileInfoPath)
```

Comment:
Will `h3_` work with `fileInfoPath` as it is? I see, that some elements can work with `Text` and some work with `String` which is a mystery for me... Why the library makes it so difficult 😞